### ISSUE TITLE
'MissingParameter' is not defined in models/dispersion_model.py

### ISSUE THREAD

**Issue by sguillot** [nanograv/PINT#480]
I had a par file with a missing DMEPOCH, resulting in a crash:
```
File "/pint/models/dispersion_model.py", line 81, in setup
    raise MissingParameter("Dispersion", "DMEPOCH",
NameError: name 'MissingParameter' is not defined
```

So, in `models/dispersion_model.py `, I think the import line
`from .timing_model import DelayComponent`
should be replaced by 
`from .timing_model import DelayComponent, MissingParameter`

**Comment by luojing1211** [comment]
Do you have DM1 in your par file? If it only DM, it should not trigger this error. Personally, I don't think this is an assumption that we should make for the users.

**Comment by sguillot** [comment]
Yes, there is a DM1.

**Comment by aarchiba** [comment]
Fixed in 886c18a4 . Please file a new bug if you want DMEPOCH to be set to PEPOCH automatically if it's missing.
